mods/default/init.lua

```diff
@@ -17,7 +17,7 @@ default.get_translator = S
 -- and avoids obscure, hard to debug runtime errors.
 -- This section should be updated before release and older checks can be dropped
 -- when newer ones are introduced.
-if ItemStack("").add_wear_by_uses == nil then
+if not minetest.features.node_interaction_actor then
 	error("\nThis version of Minetest Game is incompatible with your engine version "..
```

Comment:
`min_minetest_version` in game.conf needs updating too

Reply:
Done in 6d61b5e

Reply:
I reverted these changes as using that engine function breaks fakelib. `minetest.node_dig` does provide basic checks (though not documented), so I decided to keep using it.